### ISSUE TITLE
[Bug] Android VoiceOver does not read DetailPage inside a MasterDetailPage

### ISSUE THREAD

**Issue by HelenMamalaki** [xamarin/Xamarin.Forms#12195]
### Description
On Android tablet, VoiceOver does not read DetailPage inside a MasterDetailPage.

I am using a MasterDetailPage with MasterBehaviour set to "Split" where the MasterPage is permanently visible next to the DetailPage. If I continuously swipe for VoiceOver to read out all the elements it always skips over the DetailsPage. A similar bug was reported on iOS fixed in 4.4.0 [here
](https://github.com/xamarin/Xamarin.Forms/pull/9702) 

### Steps to Reproduce

1. Turn on voiceover 
2. Open app with MasterDetailPage, set to "Split" mode.
3. Swipe through all elements on page, you will see Details page is skipped

Things to note are that I am loading the MasterDetailPage using Prism and setting the DetailsPage (a separate content page) using Prism navigation. Hopefully that has no effect. 

### Expected Behavior
Voice over to read DetailsPage elements

### Actual Behavior
Voice over skips over DetailsPage elements, reads top and bottom menu and return back to reading MasterPage.

### Basic Information

- Version with issue: 4.5.0.356
- Last known good version: unknown
- IDE: VS2019
- Platform Target Frameworks: 
  - Android: 9.0, 10.0

**Comment by KlaasTroost** [comment]
Yesterday I installed XF 5.0.0.1558-pre3 and nothing has changed in the behavior when changing all MasterDetailPage to FlyoutPage, so the DetailPage is still not accessible.

**Comment by rachelkang** [comment]
Hi, @KlaasTroost - I believe this issue has since been fixed!

If you haven't already, I recommend updating to the latest version Xamarin.Forms or at least to Xamarin.Forms 5.0.0.2083, which has many accessibility improvements. These improvements include an experimental accessibility flag that addresses a lot of improvements in Android accessibility.

To leverage this flag on the latest versions of Xamarin Forms, the following code can be inserted in the App.xaml.cs file, following the InitializeComponent call and before the MainPage is set: `Xamarin.Forms.Device.SetFlags(new List<string> { "Accessibility_Experimental"});`

Let us know how that works for you.
